### DISCUSSION TITLE
load array?

### DISCUSSION THREAD

**Discussion by ghost** [abbaye/WPFHexEditorControl#106] [Q&A] [answered]
I saw in the sample that you pass to the control the file name ,
Can your control be used to display bytes arrays instead of load files?

**Comment by abbaye** [comment]
Yes you can use the Stream property to load bytearray. :)

You can check in sample :)

Le ven. 17 sept. 2021 3 h 18 p.m., AMT-Cheif ***@***.***> a
écrit :

**Comment by ghost** [comment]
Thanks!! 
I've see a strange thing turning the background black, the selections and highlight cover the data, but can be my mistake, I'll make some test.
There is an event handler that can be used to know when user is modding the hex data, so I can call applychange and update the stream in real time? I'll checking out myself next days but, if you have an advice it would save a lot of time :).
Thanks again!

**Comment by abbaye** [comment, marked as answer]
Hello,

Hum I need to test for background/highlights 🤔

I have not implemented a "OnByteChange" event. But it's not difficult to do
this.

If you want a can make this. But go on GitHub project to open an Issues
please :)

Best regards

Le mer. 22 sept. 2021 7 h 29 a.m., AMT-Cheif ***@***.***> a
écrit :

**Comment by ghost** [comment]
ok. thanks.
